src/fu-usb-backend.c

```diff
@@ -388,7 +388,11 @@ fu_usb_backend_get_device_parent(FuBackend *backend,
 		g_set_error_literal(error, FWUPD_ERROR, FWUPD_ERROR_NOT_SUPPORTED, "no device");
 		return NULL;
 	}
+#ifdef HAVE_LIBUSB_WRAP_SYS_DEVICE
 	usb_parent = libusb_get_parent(usb_device);
+#else
+	usb_parent = NULL;
```

Comment:
I would push this up to the declaration.

```
libusb_device *usb_parent = NULL;
```

Then over here you have the `#ifdef` but no `#else` case.